packages/sites/genomics-site/webapp/wdkCustomization/js/client/userDatasetRoutes.tsx

```diff
@@ -19,7 +19,7 @@ const UserDatasetRouter = React.lazy(
 
 const availableUploadTypes = userDatasetsUploadTypes.trim()
   ? userDatasetsUploadTypes.split(/,\s*/g)
-  : ['genelist', 'bigwigfiles', 'rnaseq'];
+  : ['genelist', 'bigwigfiles', 'rnaseq', 'wrangler'];
```

Comment:
I'm not so keen on `wrangler` being user-facing, albeit in a URL. Is this already set in stone or can we replace it?

I see that it might be a conifer thing (`userDatasetsUploadTypes` defined in packages/libs/web-common/src/config.ts)

So these are the default fallbacks?

I would call this `study` or `generic` or something? Can discuss tomorrow.